tests/data/mocha/json/multiple-suites-multiple-tests-tags.json

```diff
@@ -0,0 +1,85 @@
+{
+  "stats": {
+    "suites": 2,
+    "tests": 3,
+    "passes": 2,
+    "pending": 0,
+    "failures": 1,
+    "start": "2022-06-11T05:28:11.204Z",
+    "end": "2022-06-11T05:28:11.227Z",
+    "duration": 7
+  },
+  "tests": [
+    {
+      "title": "sample test case @fast #1255",
+      "fullTitle": "Example Suite 1 sample test case @fast #1255",
```

Comment:
I've noticed a pattern where users provide additional information about the tag in the test case description using the equals sign. For instance, `Example Suite 1 sample test case @slow @owner=john @suite=regression`. It would be great to store this information as key-value pairs under meta_data.

```js
testcase.meta_data.set("tags", "slow");
testcase.meta_data.set("owner", "john");
testcase.meta_data.set("suite", "regression");
```

Reply:
Agreed, this would be good guidance to bring keyed data for mocha and cucumber frameworks.